docs/templates/e-commerce.rst

```diff
@@ -0,0 +1,24 @@
+.. _e-commerce:
+
+===========================================
+E-commerce spider template (``e-commerce``)
+===========================================
+
+Basic use
+=========
+
+.. code-block:: shell
+
+    scrapy crawl e-commerce -a url="https://books.toscrape.com"
+
+Parameters
+==========
+
+.. autopydantic_model:: zyte_spider_templates.spiders.ecommerce.EcommerceSpiderParams
+    :inherited-members: BaseModel
```

Comment:
We need to add a `description` to https://github.com/zytedata/zyte-spider-templates/blob/main/zyte_spider_templates/spiders/ecommerce.py#L52 so we can have details on this:

![image](https://github.com/zytedata/zyte-spider-templates/assets/3449761/59ddc851-8473-4560-a325-74045a9be78c)